lib/entities/editor-interface.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * Editor Interface instances
+ * @namespace EditorInterface
+ */
+
+import cloneDeep from 'lodash/cloneDeep'
+import freezeSys from 'contentful-sdk-core/freeze-sys'
+import enhanceWithMethods from '../enhance-with-methods'
+import mixinToPlainObject from 'contentful-sdk-core/mixins/to-plain-object'
+import omit from 'lodash/omit'
+import errorHandler from '../error-handler'
+
+function createEditorInterfaceApi (http) {
+  return {
+    /**
+     * Sends an update to the server with any changes made to the object's properties
+     * @memberof EditorInterface
+     * @func update
+     * @return {Promise<EditorInterface>} Object returned from the server with updated changes.
+     * @example
+     * editorInterface.controls[0] = { "fieldId": "title", "widgetId": "singleLine"}
+     * editorInterface.update()
+     * .then(editorInterface => console.log(editorInterface.controls))
+     */
+    update: function () {
+      const raw = this.toPlainObject()
+      const data = omit(raw, ['sys'])
+      return http.put('content_types' + '/' +
+       this.sys.contentType.id +
+       '/editor_interface',
```

Comment:
I'd probably say it would be good to keep this string generation all in one line as it's not too long. You can use ES2015 template strings to make it a bit more compact.